### ISSUE TITLE
Help with direct path for plex in docker

### ISSUE THREAD

**Issue by Goldsucher** [croneter/PlexKodiConnect#1814]
Hi croneter 😊
I'm trying to setup PlexKodiConnect but I think I need some help with direct path for docker. Unfortunately docker isn't really mentioned in your wiki.
Here is my setup:

- Synology NAS with movies / tv show AND exported .strm files from Netflix / Amazon Prime Kodi Addons
- Plex Server running in Docker on my Synology NAS
- Nvidia SHIELD with Kodi 18, PlexKodiConnect and access to my NAS via smb.
- videos are accessible from Kodi via smb file share 

my file structure is:

videos/movies
videos/tvshows
videos/netflix/movies
videos/netflix/shows
videos/Amazon/Movies
videos/Amazon/TV

I can't really change this structures because I can't split the netflix / amazon subfolders (movies and tv shows) because they are created by the addons.
Plus I cannot use Customize Paths from PKC because of the file structure.

I know that I have to set direct path to get .strm files to work but I don't know how to do that with my docker setup. 

I mounted my videos folder into the docker container so the paths are /videos/movies/, /videos/netflix/movies, and so on. 
As far as I know it is not possible to mount a smb share into a docker container (and it would be quite pointless because both is running on the same machine).

Could you help me to get direct paths to work for my setup?

Thanks a lot!

**Comment by Goldsucher** [comment]
Okay after many tries I've fixed it. The solution is pretty much as it is described here:
https://github.com/croneter/PlexKodiConnect/wiki/Option-2-Substitute-Paths-on-the-PMS

Here are my steps, if anyone has the same problem:

1. completely reset the database (with resetting PKC settings): PKC Settings --> advance --> the red entry
2. after restart, choose native mode. I only entered the local ip of my NAS, without smb:// or subfolders (like: 192.168.230.2)
3. than in the next prompts enter smb, username and password
4. than go back to PKC settings --> change paths: first option unchecked, second option checked.
5. I replaced /videos/ with smb://ip-of-nas/videos on movies and tv shows 
6. reset database again (without PKC settings) and than it should work!


thanks for your plugin!
